### PR TITLE
Refactor the universe v3 code to use new client

### PR DESCRIPTION
**Pull request by jsancio**

### CODE REVIEW

File: cosmos-server/src/main/scala/com/mesosphere/cosmos/repository/UniverseClient.scala [hunk 1]
```diff
@@ -89,9 +110,6 @@ class UniverseRepositoryFetcher(
             "User-Agent",
             s"cosmos/$cosmosVersion dcos/${dcosReleaseVersion.show}"
         )
-        session.authorization.foreach { auth =>
-          conn.setRequestProperty("Authorization", auth.headerValue)
-        }
```
Comment (cruhland) [inline cosmos-server/src/main/scala/com/mesosphere/cosmos/repository/UniverseClient.scala]: Why was this deleted?

File: cosmos-server/src/main/scala/com/mesosphere/cosmos/MultiRepository.scala [hunk 2]
```diff
@@ -116,19 +122,20 @@ final class MultiRepository (
...
   override def getPackagesByPackageName(
-    packageName: String
+      packageName: String
   ): Future[List[internal.model.PackageDefinition]] = ???
 
   override def getPackageByPackageVersion(
-    packageName: String,
-    packageVersion: Option[universe.v3.model.PackageDefinition.Version]
+      packageName: String,
+      packageVersion: Option[universe.v3.model.PackageDefinition.Version]
```
Comment (cruhland) [inline cosmos-server/src/main/scala/com/mesosphere/cosmos/MultiRepository.scala]: Some extra whitespace was added here.

**Comment by cruhland** [comment]
LGTM after comments are addressed.

File: cosmos-server/src/main/scala/com/mesosphere/cosmos/repository/UniverseClient.scala (hunk 1 above)
Comment (BenWhitehead) [inline cosmos-server/src/main/scala/com/mesosphere/cosmos/repository/UniverseClient.scala]: this has to stick around so that we can communicate with admin router when it's auth is enabled.

Comment (jsancio) [inline cosmos-server/src/main/scala/com/mesosphere/cosmos/repository/UniverseClient.scala]: I think this class should only talk to Universe and should not care about DC/OS authorization. There are a few benefits to this but the main benefit is that we will never accidentally leak authorization tokens to external Universes. What do we think?

File: cosmos-server/src/main/scala/com/mesosphere/cosmos/MultiRepository.scala (hunk 2 above)
Comment (jsancio) [inline cosmos-server/src/main/scala/com/mesosphere/cosmos/MultiRepository.scala]: I see. This is my autoformatter.

File: cosmos-server/src/main/scala/com/mesosphere/cosmos/repository/UniverseClient.scala (hunk 1 above)
Comment (BenWhitehead) [inline cosmos-server/src/main/scala/com/mesosphere/cosmos/repository/UniverseClient.scala]: The auth token is only needed for admin router, not universe you are correct. I should have expanded the diff collapse to see this is for universe fetch step not the previous request to admin router to get the dcos version.
